tabs/ports.js

```diff
@@ -4,6 +4,7 @@ TABS.ports = {};
 
 TABS.ports.initialize = function (callback, scrollPosition) {
     var board_definition = {};
+    var isSupportPeripherals = semver.gte(CONFIG.apiVersion, "1.27.0");
```

Comment:
@digitalentity 
I've update it, and I'm post a  wrong comment of the last commit in this PR, in my code, I'm using FC version to detect which is support peripherals configuration or not.
But actually, the result is same.